docs/how-tos/configure.md

```diff
@@ -62,13 +65,15 @@ Each physical piece of your smart machine that is controlled by a computer is ca
 
 For each component that makes up your machine:
 
-1. Find an appropriate model for your hardware. For example, you can scroll through available sensor models on the [sensor page](/components/sensor/#available-models).
+1. Find an appropriate model for your hardware.
+   You can find the available models on the [component pages](/components/).
+   For example, you can scroll through available sensor models on the [sensor page](/components/sensor/#available-models).
 2. Add a suitable model to your machine on the **CONFIGURE** page:
 
    - Click the + icon next to your machine part in the left-hand menu and select Component.
    - [Choose any component type](/configure/#components) (example: `camera`) and model (example: `webcam`).
 
-3. Click on the **Test** area of the configuration panel to test your component.
+3. Click on the **Test** area of the configuration panel to test controlling your component.
 4. If any problems occur check the [logs](/cloud/machines/#logs) or review or roll back the [configuration history](/cloud/machines/#configure).
 
 If a component you want to use for your project is not natively supported, you can [build your own modular resource](/how-tos/create-module/).
```

Comment:
```
You need to [_configure_](/configure/) your machine so that `viam-server` can interact with its hardware.
Use the configuration builder tool in the Viam app to create a file that describes what hardware you are using and how it is connected.
For example, if you have a DC motor, follow the [corresponding configuration instructions](/components/motor/gpio/) to tell the software which pins it is connected to.
```

The following lines don't make sense anymore. They presumably should be with step 2. Meaning there should be another step before testing that says

```
1. When you add a component model, it will create a panel in the configuration builder tool.
    Fill in any required attributes, following the documentation for the specific model. 
```

Reply:
sorry, was slightly confused but I think I implemented what you want here! lmk if it looks good or if I need to make adjustments